src/lib/http/http_server.pl

```diff
@@ -68,6 +68,11 @@
 :- use_module(library(pio)).
 :- use_module(library(time)).
 
+%% http_listen(+Port, +Handlers).
```

Comment:
For example, here, `_` is not escaped (in contrast to above), and it seems far preferable to write `_` directly like this.